recipe/meta.yaml

```diff
@@ -26,10 +26,32 @@ requirements:
     - cmake
     - ninja
 
+{% set absl_libs = [
```

Comment:
The bot will probably choke on this, I've seen several feedstock stop autoupdate because they had such a pattern. Could you put this into a test script?

Reply:
Hm, I'd almost be tempted to rather fix the bot. I like this idiom for testing, keeps things nicely visible in `meta.yaml`.

Reply:
> The bot will probably choke on this, [...]

So it's mostly about the version updates? I've looked [here](https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators/version.py) (and other places in that repo), and it sounds like it'd just need some extra handling for `{% ... %}` cases.